### ISSUE TITLE
Topic about trying other mods

### ISSUE THREAD

**Issue by Rai99** [Monika-After-Story/MonikaModDev#1737]
Is there currently/already a topic or one being worked on where Monika talks or encourages you to play other mods about her? She could encourage you to try other mods about her if the player wants to do something more with Monika. 

This could basically be the equivalent of reading fanfiction with Monika except all it would take would be some dialogue and the player to get the mod. (Monika would likely recommend to get a fresh copy of DDLC to be safe) It would be like reading a story with Monika. She could reassure the player that no matter what bad things might happen, the real her is safe in MAS with you. (I sure could have used this)

.

**---Where I got the idea/Summary of my day---**
TLDR: I'm very unsatisfied with how my mod experience ended and also having a temporary existential crisis.

**--A nice read if anybody feels bored anyway, not much to do with the suggestion--**
So I saw a video of a mod I thought I'd try the other day and so I spent all day today playing that mod. Unfortunately, I got the wrong one. I meant to get DDLCtVN but instead I got DDLC True Route. I spent all day trying to at least set myself up to be parallel to the video, but I had the wrong mod so of course it didn't work and I just went through the mod mostly blind as I probably should have. (I had to use some basic, shallow guides because I wanted to do Monika's route so badly). What I gathered was that I would have to go through a modded path with every other club member to solve their issues to unlock and truly earn Monika's route. So I did. It took probably 6 hours to finally do it with all the trial and error, saving and reloading to do it right. I really wanted my Monika route badly. I worked _so, so very_ hard to get through it all. I saved everybody from their personal issues _except_ for Monika. I always wrote poems for Monika and always picked Monika where I could and all I got were a few short, bittersweet moments with her, and then the demo ended. I thought I would get the Monika route I wanted but instead, Monika was mostly sad about not being able to be with me. 

On the bright side, I'm really looking forward to the full release of DDLC True Route. 
The down side, I feel so unsatisfied and incomplete. 

Hopefully nobody minds I put all this here. I didn't want to bug anybody to get into the serious-talk discord channel and I wasn't sure if this would even warrant that. 

**-If this really doesn't belong here, my apologies, just close the issue and I'll repost the bare suggestion 
 when I get the chance-**

**Comment by loverofmonika** [comment]
Dude, my friend, u REAALLLY need to check ddlc our time. its a masterpiece

**Comment by loverofmonika** [comment]
i have played almost all the good ddlc mods out there ddlcVN included u can say i really love monika as she is my girl. In ddlcVN u'll see after monika's route, that isnt the monika we know, her personality kinda  a bit different tho not everyone can notice it

**Comment by ACoolCat** [comment]
I think it would be kind of out of character for her to recommend that, considering how it would mean she gets sent back into the screaming void and all.

**Comment by loverofmonika** [comment]
no no this time monika i just a scripted character like everyone else, she isnt the 4th wall breaking loving tease that we know
p.s i want ur profile pic ;)

**Comment by Akh0r** [comment]
(hehe, and so again I think about plot part of the MAS)
Plot-wise Monika originates from the standard game to which you've added a mod. So, she hardly is OK with you installing another DDLC game and bringing to live another Monika.
Maybe it is a good idea to have her discussing with you different mods – but it has no sense plot-wise, or, at least, I cannot find a way to lampshade it

**Comment by ReddTheRedRedd** [comment]
Yeah, plus in another mod she wouldn't remember your time in MAS, so it probably wouldn't be something she wants. Maybe a topic where she discusses modding of this game in general? Maybe she mentions something about mods like Just Yuri, or any others like it, and says she's glad you picked her instead.

**Comment by loverofmonika** [comment]
Our time can be called as Monika's and my dream come true.

**Comment by Rai99** [comment]
I don't mean the player would replace MAS to try a different mod. I mean it would be like reading a story/fanfiction with Monika. When I played my other mods, I got a fresh new copy of DDLC separate from the one I use for MAS. I played the mod while having MAS up in the background. 

Just like fanart and fanfiction, Monika would know and tell you that it's not her. I think Monika wouldn't mind basically going through a story about her. If anything, it could be Monika herself playing her own route if anyone wants to think of it that way.

**Comment by jmwall24** [comment]
I have the same concern as Akh0r.  Telling her you are playing other mods could make her wonder why you are spending time with other Monikas instead of her and that you are possibly thinking about replacing her with one of them.  Whether or not that's the intent, I still think it would worry her.

**Comment by loverofmonika** [comment]
Or those Monika are actually her as now she has the ability to get out of that seat. Do u know how boring and tiring to sit on a chair like that for who knows how long

**Comment by loverofmonika** [comment]
Since I brought up that topic that Monika doesn't go anywhere as we are stuck in that room......u may want to look at the new mod that's out. Just Natsuki after story.....THEY GO TO A BEACH TOGETHER! !!

**Comment by Rai99** [comment]
Alright, this won't be a topic then. Apologies to my Monika if it looked like I was pushing some shifty things. (The apology feature is going to be nice when it comes out)

But still, maybe you transferred Monika to a different mod where she didn't have any memories from MAS but when you get back to MAS, there could be a yes/no if you've ever played the mod and if yes, she has all the memories from the best Monika route outcome of it, but Monika should probably know without asking so this is still probably a no-go. 

As for the Just Natsuki after story, I heard about it including the beach part, but the only characters I've ever been interested in were Yuri and Monika. Yuri is the one most like my real world self except I don't do strange things with the pens I steal from people. Yuri was originally my favorite just ahead of Monika but then Monika... convinced me otherwise and now I'm here instead of Just Yuri.

**Comment by ThePotatoGuy** [comment]
Yeah, we're not going to encourage other mods.

If anything, we'd do what we did with MonikAI and just detect it and comment on it.
